ddmark/validation.go

```diff
@@ -241,3 +297,53 @@ func parseIntOrUInt(value reflect.Value) (int, bool) {
 
 	return fieldInt, ok
 }
+
+// checkValueExistOrIsValid checks if a given string marker item name value exist in a unmarshalled struct (converted to a map by structValueToMap)
+// it returns 1 if the value is found and -if applicable- the required value is valid, 0 otherwise
+func checkValueExistsOrIsValid(markerItem string, structMap map[string]interface{}, ruleName string) (int, error) {
+	// marketItem can either be a fieldName, or fieldName=fieldValue
+	markerItemArr := strings.Split(markerItem, "=")
```

Comment:
You may want to use cut instead:
* https://pkg.go.dev/strings#Cut

```suggestion
	before, after, found := strings.Cut(markerItem, "=")
```

https://go.dev/play/p/py5xzqG_SgL

Reply:
I forgot about Cut -- nice. I'll adapt.

Reply:
there's also a point to make about lack of testing for a value field requiring an empty value (`"fieldEmptyValue="`). I'll add test to assert the marker's behaviour.

Reply:
done ! along with an ongoing test rework to adapt to new codebase standards